<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectResearch</class>
 <widget class="QWidget" name="SelectResearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Research Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0" columnstretch="1,1,1,1,1">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="startDate">
       <property name="text">
        <string>Start date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QLabel" name="endDate">
       <property name="text">
        <string>End date:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QListWidget" name="checklist"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="sensorButton">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="researchTitle">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QProgressBar" name="researchProgress">
       <property name="toolTip">
        <string>Research progress</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Sensors</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Progress:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QTextBrowser" name="researchInfo">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Information:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Checklist:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="logoutButton">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sensorButton</tabstop>
  <tabstop>checklist</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>logoutButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
